pkg/rdt/rdt.go

```diff
@@ -87,6 +90,30 @@ type MonGroup interface {
 	Parent() CtrlGroup
 }
 
+// Monitor is the interface for retrieving RDT monitoring data
+type Monitor interface {
+	GetMonData() MonData
+}
```

Comment:
This is just a nitpicking question out of curiosity: the `Monitor` interface is never used anywhere as such, not even in test cases. IOW, practically every `ResctrlGroup` is `Monitor`able and currently everything/the only thing `Monitor`able is a `ResctrlGroup`. Is there any real/particular reason for splitting `Monitor` out as a separate interface, maybe some future changes in mind ?

Reply:
Ach, another good catch 👍 A stranded leftover interface from some earlier version. I'll drop it

Reply:
Fixed: I dropped the Monitor interface.